server/db/models/orders.js

```diff
@@ -6,8 +6,10 @@ module.exports = function (db) { //two files, one for order, one for orderDetail
     db.define('order', {
         address: {
             type: Sequelize.TEXT, //no empty strings? CdV/OB
-            allowNull: false,
-            validate: {notEmpty: true}
+            // unable to add to order as unregistered user if address is required
+            // at creation; should replace allowNull with a validation at order placement
+            // allowNull: false,
+            // validate: {notEmpty: true}
         },
```

Comment:
cool - agreed.  this validation should only exist if status is 'active' (at checkout)